src/frontend/src/App.js

```diff
@@ -128,7 +128,7 @@ function App() {
       });
   };
 
-  const handlSaveAnnotation = async () => {
+  const handleSaveAnnotation = () => {
```

Comment:
Did you mean to remove the `async`? This seems fine to me, I just wanted to double check that it was intentional.

Reply:
`async` seems to me unnecessary here because we don’t use it in the similar functions anywhere else.